aiohttp/client.py

```diff
@@ -218,6 +227,18 @@ def _request(self, method, url, *,
         handle = tm.start()
 
         url = URL(url)
+
+        if trace_context is None:
+            trace_context = SimpleNamespace()
+
+        yield from self.on_request_start.send(
+            trace_context,
+            method,
+            url.host,
```

Comment:
The whole URL object maybe? Query part might be interested for tracer.

Reply:
Yes, I will do that